docs/theatre_cli/config.md

```diff
@@ -0,0 +1,24 @@
+# `theatre_cli/config.py`
+
+The `config.py` module is responsible for managing configuration settings for your Theater CLI application. It handles loading and saving user settings, such as the movie directory and first-time setup.
+
+## Functions:
+1. `load_config()`
+    * Description: Loads the configuration from a file named "config.json." It returns the configuration data as a list of dictionaries.
+    * Usage:
+        ```python
```

Comment:
In the entire documentation, can you give one line space before writing a code block for usage?

```suggestion
    * Usage:
    
        ```python
```